show_menu.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'app'
+def show_menu_list
+  puts ''
+  puts 'Welcome to School Library App!'
+  puts ''
+  puts 'Choose an option by entering a number:'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List rentals for a person'
+  puts '7. Quit'
+end
+
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength
+def show_menu
+  app = App.new
+  loop do
+    show_menu_list
+    choice = gets.chomp.to_i
+    case choice
+    when 1
+      app.books
+    when 2
+      app.people
+    when 3
+      app.add_person
+    when 4
+      app.add_book
+    when 5
+      app.add_rental
```

Comment:
- [ ] In the `app.rb` file, all codes responsible for receiving inputs from the users on the console app or codes responsible for validating the input from the user should be put in the `main.rb` file or a separate class, class method or module. The other methods for creating a class instance or listing instances should only create those instances. 👍 😎 

---

For example, the `add_book` method should only create a new instance of the `Book` class while codes for getting the `title` and `author` of the _**book**_  from the console app should be put in a separate method, class or module. 💻 

--- 

Kindly apply these changes for the following methods in the app.rb file:

- `add_teacher`
- `add_student`
- `add_person`
- `add_book`
- `add_rental`

---

#### N.B: Please ensure that the app works as it should after refactoring the code. 👍